src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment.component.ts

```diff
@@ -78,7 +78,7 @@ export class FileUploadAssessmentComponent implements OnInit, OnDestroy {
         private router: Router,
         private route: ActivatedRoute,
         private resultService: ResultService,
-        private fileUploadAssessmentsService: FileUploadAssessmentsService,
+        private fileUploadAssessmentsService: FileUploadAssessmentService,
```

Comment:
nit, I see that the name is sometimes `fileUploadAssessmentService` (without the s), I think it would be good to be consistent here, and you changed the type to the version without "s" to it would fit better anyways.

Reply:
Agree to make it consistent. Did you change every occurance of the `fileUploadAssessmentsService` type?

Reply:
Ah, yea  refactored the type, but it seems like i forgot to refactor all the places where the variable name with the 's' is used. I'll do that too.